common/onepanel/base/vars.yaml

```diff
@@ -21,9 +21,26 @@ database:
     required: true
     default: postgres
 application:
-  webHost:
+  host:
     required: true
     default: localhost
-  webPort:
-    required: true
-    default: 80
\ No newline at end of file
+  local:
+    webHTTPPort:
+      required: true
+      default: 4200
```

Comment:
These are all node ports, so they should be in the 30000-32767 range. I suggest:
```
apiHTTPPort: 32000
apiGRPCPort: 32001
webHTTPPort: 32003
```

Reply:
That's a good point, will do.